platforms/hyperledger-fabric/charts/generate_cacerts/Chart.yaml

```diff
@@ -6,6 +6,6 @@
 
 apiVersion: v1
 appVersion: "2.0"
-description: A Helm chart for ca server configuration
+description: "Hyperledger Fabric: Configures ca server."
```

Comment:
This should be a job to generate CA Server certs

Reply:
Fixed.